nextflow.config

```diff
@@ -55,16 +56,15 @@ params {
   publish_dir_mode = 'copy'
 
   // Boilerplate options
-  name = false
   multiqc_config = false
   email = false
   email_on_fail = false
   max_multiqc_email_size = 25.MB
   plaintext_email = false
   monochrome_logs = false
   help = false
-  params.project = false
-  params.clusterOptions = false
```

Comment:
Oops! 🤦 

@KevinMenden maybe we should make `params.params` another special case that is not allowed like the Nextflow cli flags..

Reply:
Well this is unexpected...

`nextflow.config`
```nextflow
params {
    foo = 'bar'
    params.baz = 'qux'
}
```

```console
$ nextflow config . -flat
params.foo = 'bar'
params.baz = 'qux'
```

Reply:
So it was actually probably working fine before, despite it being an error 😅 Ah well!

Reply:
If you use the dot notation the `params.baz` gets interpreted as the `baz` in the `params` scope